### PR TITLE
Add conda-forge recipe for sos-papermill

### PR DESCRIPTION
**Pull request by BoPeng**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by BoPeng** [comment]
@conda-forge/help-python This recipe is ready for review.

File: recipes/sos-papermill/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - nose
+    - pytest
+    - nbformat
+    - mock
+  imports:
+    - sos_papermill
+
+about:
+  home: https://github.com/vatlab/sos-papermill
+  license: BSD
```
Comment (scopatz) [inline recipes/sos-papermill/meta.yaml]: This should be something like "BSD-3-Clause"

Comment (BoPeng) [reply recipes/sos-papermill/meta.yaml]: fixed

**Comment by BoPeng** [comment]
Thanks. #9345 (sos-matlab) would be the last one left, unless work on iruby (#9380) can be resumed so that `sos-ruby` could be added.

**Pull request merged**